spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
+ActiveRecord::Migration.maintain_test_schema!
```

Comment:
If you add it in a separate PR, I'll accept and merge. Then we can merge master back here and it's separated.